src/modules/task/task.controller.ts

```diff
@@ -12,29 +12,30 @@ import { DefaultStringPipe } from '../../validators/string';
 export class TaskController {
   constructor(private taskService: TaskService) {}
 
-  @Get('all')
+  @Get(['', 'all'])
   async listAll(
     @Query() pagination: Pagination,
     @Query('search', new DefaultStringPipe()) search: string,
     @Query('user', new OptionalParseIntPipe()) user: number,
     @Query('project', new OptionalParseIntPipe()) project: number,
+    @Query('story', new OptionalParseIntPipe()) story: number,
   ) {
-    return (await this.taskService.listAll(pagination, search, user, project)).map(
+    return (await this.taskService.listAll(pagination, search, user, project, story)).map(
       task => new DTask(task),
     );
   }
 
-  @Post('create')
+  @Post(['', 'create'])
   async createTask(@Body() data: VTask) {
     return new DTask(await this.taskService.createTask(data));
   }
 
-  @Put('update')
+  @Put(['', 'update'])
   async updateTask(@Body() data: VTaskOpt) {
     return new DTask(await this.taskService.updateTask(data));
   }
 
-  @Delete('delete')
+  @Delete(['', 'delete'])
```

Comment:
Kaj je fora tega?

Reply:
Videl sem, da smo najprej začeli z endpointi tipa POST `/project/create`, kasneje pa dodali endpointe tipa POST `/sprint`. Zato sem hotel še te endpointe migrirat na ta tadrugi tip, sam dokler se ne migrira klice iz frontenda bi pustil obojne delujoče...

Kakorkoli, lahko se tudi ne ubadamo s temi stvarmi in endpointe pustim kot so in to zbrišem ven :)

Reply:
A to če jih daš v array pomeni da deluje /sprint/delete in /sprint? 
to nism vedu :D could be useful in the future :D

Reply:
ja, tako delujeta oba hkrati :)